.github/ISSUE_TEMPLATE/5_cdn.yml

```diff
@@ -0,0 +1,51 @@
+name: CND Issues
+description: Use this template to report issues with the CDN.
+labels:
+  - type::hosting
+body:
+  - type: textarea
+    id: channels
+    attributes:
+      label: Channel(s)
+      description: List the affected channel(s) (e.g., conda-forge, bioconda).
+    validations:
+      required: true
+  - type: input
+    id: date
+    attributes:
+      label: When
+      description: Provide when the problem was first noticed.
+    validations:
+      required: true
+  - type: dropdown
+    id: type
+    attributes:
+      label: Problems
+      description: Select the type of problem you are reporting.
+      multiple: true
+      options:
+        - repodata completeness
+        - package download speed
+        - channel availability / accessibility
+        - other
+    validations:
+      required: true
+  - type: textarea
+    id: packages
+    attributes:
+      label: Packages
+      description: Example packages or URLs to investigate.
+  - type: textarea
+    id: curl_output
+    attributes:
+      label: Output of `curl -I EXAMPLE_URL`
+      description: Please provide the output of the `curl` command for the impacted packages.
+  - type: dropdown
+    id: urgency
+    attributes:
+      label: Urgency
+      description: Select the urgency level of this issue.
+      options:
+        - High (will impact many users)
+        - Medium (will impact some users)
+        - Low (will impact few users)
```

Comment:
We can leave it for now, but I'm not so convinced that "urgency" is a field that can be meaningfully filled out by the reporter, with some exceptions. Mostly because it's _really_ hard to judge how many users of anaconda.org are impacted, given its reach.

Reply:
I also feel like this is redundant since we already have the severity labels